astro/src/content/articles/identity-basics/hosted-login-vs-api-login.mdx

```diff
@@ -0,0 +1,53 @@
+---
+publish_date: 2024-08-21
+title: Hosted Login Pages Versus API Login - Which Is Right For You?
+description: Compare hosted login pages and API login methods, exploring their pros and cons to help you choose the best authentication solution for your application.
+section: Identity Basics
+authors: Brad McCarty
+icon: /img/icons/hosted-vs-api.svg
+darkIcon: /img/icons/hosted-vs-api-dark.svg
+---
+
+If you're reading this then you already know about the importance of authentication. What you might not know is whether you want to use FusionAuth's hosted login pages, or if you'd prefer to use the API options instead. We put a guide together for you to help you make up your mind.
+
+We're going to explore the differences between hosted login pages and the results that you can get using the API. We will dive into their pros and cons so that you can make an informed decision about what is best for you.
+
+## Hosted Login Pages
+
+Hosted login pages are a popular choice for many developers. This is especially true for those who prefer to use a WYSIWYG editor to handle their customization. That said, Hosted Logins do require some tradeoffs such as a browser redirect. Further, since the Hosted Login is designed in our Simple Themes editor or inside of Apache FreeMarker, they do not leverage React or other common design systems.
+
+![A diagram showing Apache FreeMarker side by side with FusionAuth's Simple Theme editor.](/img/articles/hosted-vs-api/simple-vs-freemarker.png)
+
+|Pros                                   |Cons                                   |
+|-----                                  |-----                                  |
+|Managed by experts.|Look and feel customization uses Freemarker unless you use Simple Themes.|
+|No sensitive credentials seen by your application.|Browser redirect is required.|
+|Fully custom look and feel.|Mobile devices use the system browser, not native UX components. See the image below this chart for an example.|
+|Handles multiple use cases (account creation, MFA prompting, etc.)|Does not leverage React or other common design systems. Might require additional work from your design team.|
+|Includes single sign-on (SSO) between different apps.|Upgrades can be problematic unless you're using Simple Themes.|
+|Localization support.|Limited workflow customization.|
+|New workflows added regularly so you gain function without added development work.||
+|Doesn't exclude API usage. You can add custom logic such as reidrects or requiring MFA for certain groups.||
+
+### Example Hosted Login Page on Mobile
+The image below shows the mobile login experience for [Audacy](https://audacy.com). As discussed in the preceeding chart, you can see the address bar and UX components that are from the system browser, rather than the native UX components that your customers will see on a desktop experience.
```

Comment:
also the last line of this sentence looses me a bit.  You refer to "desktop experience" but are talking about mobile. I think it should be something like "...that your customers will see in your application." or something like that.